README.md

```diff
@@ -1,53 +1,55 @@
-# kwiz Ionic
+# Kwizapp Ionic React Frontend
 
-## How to Run
+<p align="center">
+  <a href="https://kwizapp.github.io" target="blank"><img src="public/assets/kwiz_logo.png" width="150" alt="Kwizapp Logo" /></a>
+</p>
 
-1. Install the Ionic CLI
+## Development
 
-    ```bash
-    npm install -g @ionic/cli
-    ```
+install npm packages
 
-2. Build tailwind styles
+```bash
+npm i
+```
 
-    ```bash
-    # once
-    npm run build:style
+### Styles
 
-    # watch mode
-    npm run dev:style
-    ```
+Build [tailwind](https://tailwindcss.com/) styles
 
-3. Start Poster API (see corresponding README):
+```bash
+# once
+npm run build:style
 
-    ```bash
-    npm run dev
-    ```
+# watch mode
+npm run dev:style
+```
 
-4. Start Metadata API (see corresponding README):
+### Start Application
 
-    ```bash
-    npm run dev
-    ```
+This is the frontend of the `kwizapp`. It depends on a few services (APIs) in order to work properly:
 
-5. Start Nest API (see corresponding README):
+- [`poster-service`](https://github.com/kwizapp/poster-service): provides movie poster image urls for IMDb ids
+- [`metadata-service`](https://github.com/kwizapp/metadata-service): provides movie metadata for IMDb ids
+- [`nest-api`](https://github.com/kwizapp/nest-api): main graphql API (get queried by the frontend)
 
-    ```bash
-    npm run dev
-    ```
+1. Start Poster API (see corresponding [README](https://github.com/kwizapp/poster-service))
+2. Start Metadata API (see corresponding [README](https://github.com/kwizapp/metadata-service))
+3. Start Nest API (see corresponding [README](https://github.com/kwizapp/nest-api))
+4. Start Ionic React Frontend with `npm run dev`
+5. Open app on PORT **3000** - http://localhost:3000/
 
-6. Start developing with
+### Component Documentation
 
-    ```bash
-    ionic serve
-    ```
+The folder `styleguide` contains automatically generated documentation of all our react components.
```

Comment:
Is this the npm script for that `styleguide:build`? When is it automatically generated, with the CI hook before comitting or manually?
Should we add it here so that it is clear how to regenerate the doc?

Reply:
It is only built when you run `npm run styleguide:build`

Reply:
added in eb97b1b

resolving